### DISCUSSION TITLE
Prepdocs --removeall not updating index

### DISCUSSION THREAD

**Discussion by IainD925** [microsoft/AI-Chat-App-Hack#69] [Q&A] [answered]
Hi.  When I run ./scripts/prepdocs**.ps1** in PowerShell v7 the process correctly processes my PDFs
However when I update ./scripts/prepdocs**.sh** with **--removeall** as shown on line 69:
`./scripts/.venv/bin/python ./scripts/prepdocs.py './data/*' --removeall --verbose \`
and then run ./scripts/prepdocs**.ps1** the index is not updated.
Can anyone advise why?

**Comment by pamelafox** [comment, marked as answer]
What do you see in the logs when you add --removeall ?

And sorry, are you updating the sh or ps1? You should update whichever one you use on your OS.

**Reply by IainD925** [reply]
Updating the .sh although I did wonder if it should be the ps1
Was going (somewhat blindly) from the video.
Now working after updating 
`$argumentList = "./scripts/prepdocs.py $dataArg --removeall --verbose " + ``

Thanks for clarifying.

**Reply by pamelafox** [reply]
Ah okay yeah, sorry, it's confusing as we have to support both ps1 and sh for different OSes, but I always use sh since I'm on a Mac. Glad it works now!
